src/server/scripts/Spells/spell_hunter.cpp

```diff
@@ -1229,6 +1230,71 @@ class spell_hun_viper_attack_speed : public SpellScriptLoader
     }
 };
 
+// 56841 - Glyph of Arcane Shot
+class spell_hun_glyph_of_arcane_shot : public SpellScriptLoader
+{
+public:
+    spell_hun_glyph_of_arcane_shot() : SpellScriptLoader("spell_hun_glyph_of_arcane_shot") {}
+
+    class spell_hun_glyph_of_arcane_shot_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_hun_glyph_of_arcane_shot_AuraScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            return ValidateSpellInfo({ SPELL_HUNTER_GLYPH_OF_ARCANE_SHOT, SPELL_HUNTER_VICIOUS_VIPER });
```

Comment:
Why do you validate SPELL_HUNTER_VICIOUS_VIPER when you do not use it within the script?